packages/flow-core/src/components/f-pictogram/f-pictogram.ts

```diff
@@ -155,9 +163,49 @@ export class FPictogram extends FRoot {
 		if (this.autoBg) {
 			return `color:${this.textColor} !important`;
 		}
+
 		return "";
 	}
 
+	get statesBasedBackground() {
+		const defaultColor = {
+			background: "var(color-neutral-sublte)",
+			hover: "var(color-neutral-sublte-hover)"
+		};
+		if (this.state) {
+			const mapper = {
```

Comment:
declare this object outside of class, it is getting created multiple time

Reply:
its dependent on `this.category`, can't declare it outside class